client/packages/common/src/ui/components/modals/InputModal/InputModal.stories.tsx

```diff
@@ -25,27 +28,39 @@ export const BasicExample = () => {
         Input={
           <BasicTextInput autoFocus onChange={e => setVal(e.target.value)} />
         }
-        onChange={() => alert(val)}
+        onChange={async () => {
+          await delay();
+          alert(val);
+        }}
       />
     </Box>
   );
 };
 
 export const NonNegativeNumberInputModalExample = () => {
   const modalController = useToggle();
+  const [val, setVal] = useState<number>();
```

Comment:
this can be removed - there's no point keeping it, now that the component has been deleted

Reply:
Have renamed to just "NumberInputModalExample"